### PR TITLE
Utils chapter 4

### PR DESCRIPTION
**Pull request by TiaanPeens**
Added chapter 4 functions to utils and removed the Chapter4.py folder

### CODE REVIEW

**Comment by TiaanPeens** [comment]
Is the build failing due to the removed file?

**Comment by TiaanPeens** [comment]
Think I found the problem

File: robustcontrol/utils.py [hunk 1]
```diff
@@ -2340,6 +2340,19 @@ def minimal_realisation(a, b, c):
...
+    Parameters: A => state space matrix A
+                B => state space matrix B
+                C => state space matrix C
+
+    Returns: is_min_real => True if the system is the minimum realisation
+    """
+    state_obsr, out_pole_vec, observe_matrix, vr = state_observability(A, C)
+    state_control, in_pole_vec, control_matrix = state_controllability(A, B)
+
+    return state_control, state_obsr
```
Comment (alchemyst) [inline robustcontrol/utils.py:2355]: These returns don't match the documentation

**Review by alchemyst — CHANGES_REQUESTED** [review]
Looks like a small issue in the documentation matching the function returns

File: robustcontrol/utils.py (hunk 1 above)
Comment (TiaanPeens) [reply robustcontrol/utils.py]: I added to check if it's only true or false.

**Pull request merged**